webapp/src/dogma/features/project/Projects.tsx

```diff
@@ -39,9 +39,9 @@ export const Projects = () => {
                 <TableCaption>Projects</TableCaption>
                 <Tbody>
                   {projects.map((project) => (
-                    <Tr>
+                    <Tr key={project.name}>
```

Comment:
🙇‍♂️

Reply:
Assuming that the name is unique ...

Reply:
Project names are unique as per the current specification.